src/lib/content.js

```diff
@@ -158,20 +159,30 @@ exports.matchesHeader = function (httpReq, specReq, ignoreHeaders) {
         const reqValue = httpReq.headers[headerName];
         if (specHeader.type) {
```

Comment:
So the use case for this is that json will never have a type added to it?

Reply:
Yes, I was thinking about adding a new type (json) but this would require changing existing contracts as well as other possible changes. So the json header should work in the same manner as the string header works right now:
1. if we have a type on a header in fixture it's just a type match, value is ignored
2. If not and the fixture header contains parsable json string, we use new logic to try and parse the request header.
3. If the fixture header is not a parsable json we proceed with the string value comparison.